From 9c0d6a61bab32592a7772c1c3085d4ff0bda493b Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Mon, 31 Oct 2022 11:54:41 +0800
Subject: [PATCH] Revert "Linux 5.10.151"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit c34d1b22fef329d5cecd003d7be249937ec70a3c.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 0e22d4c8bc79..0ecf8333cc17 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 151
+SUBLEVEL = 150
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

